sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -0,0 +1,1188 @@
+  ): Promise<ServiceGetUserDelegationKeyResponse>;
+  // (undocumented)
+  listFileSystems(
+    options?: ServiceListFileSystemsOptions
+  ): PagedAsyncIterableIterator<FileSystemItem, ServiceListFileSystemsSegmentResponse>;
+}
+
+export { deserializationPolicy };
+
+// @public (undocumented)
+export interface FileAppendOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  conditions?: LeaseAccessConditions;
+  // (undocumented)
+  contentHash?: Uint8Array;
+}
+
+// @public (undocumented)
+export interface FileFlushOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  close?: boolean;
+  // (undocumented)
+  conditions?: DataLakeRequestConditions;
+  // (undocumented)
+  pathHttpHeaders?: PathHttpHeaders;
+  // (undocumented)
+  retainUncommittedData?: boolean;
+}
+
+// @public (undocumented)
+export interface FileReadHeaders {
+  // (undocumented)
+  acceptRanges?: string;
+  // (undocumented)
+  blobCommittedBlockCount?: number;
+  // (undocumented)
+  blobContentMD5?: Uint8Array;
+  // (undocumented)
+  blobSequenceNumber?: number;
+  // (undocumented)
+  cacheControl?: string;
+  // (undocumented)
+  clientRequestId?: string;
+  // (undocumented)
+  contentCrc64?: Uint8Array;
+  // (undocumented)
+  contentDisposition?: string;
+  // (undocumented)
+  contentEncoding?: string;
+  // (undocumented)
+  contentLanguage?: string;
+  // (undocumented)
+  contentLength?: number;
+  // (undocumented)
+  contentMD5?: Uint8Array;
+  // (undocumented)
+  contentRange?: string;
+  // (undocumented)
+  contentType?: string;
+  // (undocumented)
+  copyCompletedOn?: Date;
+  // (undocumented)
+  copyId?: string;
+  // (undocumented)
+  copyProgress?: string;
+  // (undocumented)
+  copySource?: string;
+  // (undocumented)
+  copyStatus?: CopyStatusType;
+  // (undocumented)
+  copyStatusDescription?: string;
+  // (undocumented)
+  date?: Date;
+  // (undocumented)
+  encryptionKeySha256?: string;
+  // (undocumented)
+  etag?: string;
+  // (undocumented)
+  isServerEncrypted?: boolean;
+  // (undocumented)
+  lastModified?: Date;
+  // (undocumented)
+  leaseDuration?: LeaseDurationType;
+  // (undocumented)
+  leaseState?: LeaseStateType;
+  // (undocumented)
+  leaseStatus?: LeaseStatusType;
+  // (undocumented)
+  metadata?: Metadata;
+  // (undocumented)
+  requestId?: string;
+  // (undocumented)
+  version?: string;
+}
+
+// @public (undocumented)
+export interface FileReadOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  conditions?: DataLakeRequestConditions;
+  // (undocumented)
+  maxRetryRequests?: number;
+  // (undocumented)
+  onProgress?: (progress: TransferProgressEvent) => void;
+  // (undocumented)
+  rangeGetContentCrc64?: boolean;
+  // (undocumented)
+  rangeGetContentMD5?: boolean;
+}
+
+// @public (undocumented)
+export type FileReadResponse = FileReadHeaders & {
+  blobBody?: Promise<Blob>;
```

Comment:
For storage-blob we renamed `blobBody` to `contentAsBlob to avoid confusion with Storage Blob. Maybe rename here too for consistency.

Reply:
Will do